templates/lzv-rights/protection.jade

```diff
@@ -0,0 +1,386 @@
+			#quiz_application
+				br
+				div(v-if='!started')
+				div(v-else-if='progress')
+					.quiz_container.quiz
+						.quiz_progess
+							| Питання:
+							span  {{index + 1}} / {{quiz.length}}
+						.quiz_question
+							h3(v-html='quiz[index].title')
+							div(v-if='quiz[index].description !== false')
+								p {{quiz[index].description}}
+							div(v-if='quiz[index].img !== false')
+								img(v-bind:src='quiz[index].img')
+						.quiz_answers(v-for='(answer, key) in quiz[index].answers')
+							.quiz_answer
+								div(v-if='quiz[index].type == 0')
+									button.quiz_radiobutton(v-if='answers[index] !== key', @click='choose(key)')
+									button.quiz_radiobutton.chose(v-else='')
+								div(v-else='')
+									button.checkbox(v-if='answers[index][key] !== true ', @click='choose(key)')
+									button.checkbox(v-else='', @click='choose(key)')
+										svg(width='16', height='11', viewBox='0 0 16 11', fill='none', xmlns='http://www.w3.org/2000/svg')
+											path(d='M6.04309 10.943L0.0608365 5.33851C-0.0202788 5.26252 -0.0202788 5.14853 0.0608365 5.07254L1.78454 3.45769C1.86565 3.38169 1.98733 3.38169 2.06844 3.45769L6.18504 7.31433L13.9316 0.0569948C14.0127 -0.0189983 14.1343 -0.0189983 14.2155 0.0569948L15.9392 1.67185C16.0203 1.74784 16.0203 1.86183 15.9392 1.93782L6.327 10.943C6.24588 11.019 6.12421 11.019 6.04309 10.943Z', fill='#65B2E1')
+								.quiz_margin_left
+									p(v-if='!answer.image') {{answer.body}}
+									img(v-else='', v-bind:src='answer.image')
+									p.quiz_hint.true(v-if='quiz[index].type == 0 && submit == true && answers[index] === key && answer.correct === true') {{answer.hint}}
+									p.quiz_hint.true(v-if='quiz[index].type == 1 && submit == true && answers[index][key] == true &&  answer.correct === true') {{answer.hint}}
+									p.quiz_hint.false(v-if='quiz[index].type == 0 && submit == true && answers[index] === key && answer.correct === false') {{answer.hint}}
+									p.quiz_hint.false(v-if='quiz[index].type == 1 && submit == true && answers[index][key] == true &&  answer.correct === false') {{answer.hint}}
+						button.quiz_button.disabled(v-if='chosen == 0 && submit === false') Відповісти 
+						button.quiz_button(v-else-if='chosen != 0 && submit === false', @click='submitAnswer()') Відповісти 
+						.quiz_main_hint(v-else-if='submit === true')
+							button.quiz_button(@click='goNext()') Далі
+							div
+								p(v-if='check == true', v-html='quiz[index].true_hint')
+								p
+								p(v-if='check == false', v-html='quiz[index].false_hint')
+								p
+				div(v-else='')
+					.quiz_container.result
+						h5 Ваш результат
+						h5
+						h2 {{score}}/{{answers.length}}
+						p {{resultText[score].description}}
+						button.quiz_button(@click='restart()') Скласти ще
+		
+		
+		script(src='https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js')
+		script.
+			new Vue({
+			el: '#quiz_application',
+			data: function () {
+			return {
```

Comment:
Maybe we should make separate `.js` file for that? Maybe even create a separate `Vue` application (using `vue-cli` e.g.) and place minified `js` files, because it will render `HTML` code also.
The whole usage above is the same.

Reply:
yah, this whole pull request is a huge salad. i hate it